epub33/core/index.html

```diff
@@ -6101,14 +6182,26 @@ <h4>URLs in the OCF Abstract Container</h4>
 &lt;/html></pre>
 					</aside>
 
-					<p class="note"> The properties of the <a>container root URL</a> are such that whatever the amount
-						of <a data-cite="url/#double-dot-path-segment">double-dot path segments</a> in a URL string (for
-						example, <code>../../../secret</code>), it will be parsed to a content URL (and not "leak"
-						outside the container). However, for better interoperability with non-conforming or legacy
-						Reading Systems, EPUB Creators should avoid using more <a
-							data-cite="url/#double-dot-path-segment">double-dot path segments</a> than needed to reach
-						the target container file. </p>
-				</section>
+					<aside class="example" title='An "out-of-container" URL'>
+						<p>Given the following container structure:</p>
+
+						<pre>
+							/
+							├── mimetype
+							├── META-INF
+							│   └── container.xml
+							└── EPUB
+							    └── content.xhtml
+						</pre>						
+
+						<p>
+							The URL `../../../../EPUB/secret.xhtml`, appearing in `content.xhtml`, is out of container
+							and would be flagged as non acceptable by a checker tool based on the relevant <a href="algo-out-of-container">algorithm</a>. Furthermore, following the 
+							<a href="#confreq-root-url">constraints</a> on the <a>container root URL</a> the Reading System parses
+							this relative URL into <a>content URL</a> with a path `EPUB/secret.xhtml`.
```

Comment:
I would present this the other way around, something like:

> A URL `../../../../EPUB/secret.xhtml` appearing in content.xhtml would be parsed by a Reading System into a content URL with a path `EPUB/secret.xhtml`, following the constraints on the container root URL. However, as the URL could be perceived as one of a resource outside the container, and create interoperability issues, it would be reported as a warning by a checker tool.